internal/providers/github/common.go

```diff
@@ -564,6 +564,32 @@ func (c *GitHub) CreatePullRequest(
 	return pr, nil
 }
 
+// ClosePullRequest closes a pull request in a repository.
+func (c *GitHub) ClosePullRequest(
+	ctx context.Context,
+	owner, repo, number string,
+) error {
+	u := fmt.Sprintf("repos/%v/%v/pulls/%v", owner, repo, number)
```

Comment:
Instead of hand-crafting the close you can use PullRequest.Edit and set the state to closed I think

Reply:
ah, missed that, thanks! 👍